httpclient_android/app/src/main/java/com/backblaze/b2/client/webApiHttpClient/android/MainActivity.java

```diff
@@ -0,0 +1,14 @@
+package com.backblaze.b2.client.webApiHttpClient.android;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class MainActivity extends AppCompatActivity {
```

Comment:
is this file needed?

Reply:
It's an example Android App where integration tests/example can be added.